messaging/infinispan-grpc-kafka/src/main/java/io/quarkus/ts/messaging/infinispan/grpc/kafka/providers/SaslSslKafkaProvider.java

```diff
@@ -0,0 +1,85 @@
+package io.quarkus.ts.messaging.infinispan.grpc.kafka.providers;
+
+import java.io.File;
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Properties;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.inject.Produces;
+import jakarta.inject.Named;
+
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.KafkaAdminClient;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.common.config.SaslConfigs;
+import org.apache.kafka.common.config.SslConfigs;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+public class SaslSslKafkaProvider extends KafkaProviders {
+
+    private final static String SASL_USERNAME_VALUE = "client";
+    private final static String SASL_PASSWORD_VALUE = "client-secret12345678912345678912";
+
+    @ConfigProperty(name = "kafka.ssl.truststore.location")
+    Optional<String> trustStoreFile;
+
+    @ConfigProperty(name = "kafka.ssl.truststore.password")
+    Optional<String> trustStorePassword;
+
+    @ConfigProperty(name = "kafka.ssl.truststore.type")
+    Optional<String> trustStoreType;
+
+    @ConfigProperty(name = "kafka-client-sasl-ssl.bootstrap.servers")
+    Optional<String> saslSslKafkaBootStrap;
+
+    @ApplicationScoped
+    @Produces
+    @Named("kafka-consumer-sasl-ssl")
+    KafkaConsumer<String, String> getSaslSslConsumer() {
+        Properties props = setupConsumerProperties(saslSslKafkaBootStrap.get(), "test-consumer");
```

Comment:
At this point `setupConsumerProperties` method ignore groupId, we have it hardcoded as `test-consumer`. So let's drop this parameter or replace with passed groupId variable